themes/classic/_dev/js/cart.js

```diff
@@ -169,27 +169,25 @@ $(document).ready(() => {
   };
 
   //Hide 'have a code promo' text if displayed when 'close' is
-  let isPresent = $('.promo-code-button').length,
+  let promoButtonsNb = $('.promo-code-button').length,
       nameLength = 0;
-  
-  $('.cancel-promo').hide();
 
-  function labelCodeDisplayed() {
+  function initPromoCodeButtons() {
     $('.cancel-promo').on('click', (e) => {
       $(e.currentTarget).hide();
       $('.promo-code-button:not(.cancel-promo)').show();
     });
 
     $('.promo-code-button:not(.cancel-promo)').on('click', (e) => {
         $(e.currentTarget).hide();
-        $('.cancel-promo').show();
+        $('.cancel-promo').css('display','block');
```

Comment:
Isn't it what `show()` does?

Reply:
Yes but the target is an inline element so I had to force with 'display: block' to make it look like it was before